### PR TITLE
StickyButton Added

### PR DESCRIPTION
**Pull request by simran-kaurbrar**
- Button is added to all pages
- Made component of skickyButton and imported in App.jsx

### CODE REVIEW

File: src/main.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import { BrowserRouter } from 'react-router-dom';
-import App from 'src/App.jsx';
+import App from './App.jsx';
```
Comment (mgao08) [inline src/main.jsx:4]: Why changed to relative path?

File: src/App.jsx [hunk 2]
```diff
@@ -13,6 +14,7 @@ const App = () => {
 		<div>
 			<Navbar />
 			<Router />
+			<StickyButton />
```
[the remark this answers is not in the document]
Comment (simran-kaurbrar) [reply src/App.jsx:17]: It was showing me error that's why i changed this. 
So, i need to close this request and update the files again?

File: src/main.jsx (hunk 1 above)
Comment (simran-kaurbrar) [reply src/main.jsx:4]: It was showing me error that's why i changed this.

File: src/App.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (mgao08) [reply src/App.jsx:17]: No just do another commit & push with your edits on this branch

File: src/main.jsx (hunk 1 above)
Comment (mgao08) [reply src/main.jsx:4]: I see, but its not a good idea to use relative path here. Maybe try  add another '/' before the src? So make the path `/src/App.jsx` and see if that works

**Pull request closed without merging**